source/dagger/gameplay/ping_pong/ball_glowing.cpp

```diff
@@ -0,0 +1,31 @@
+#include "ball_glowing.h"
+#include "core/engine.h"
+#include "core/graphics/sprite.h"
+
+#include "gameplay/common/simple_collisions.h"
+
+void BallGlowingSystem::Run()
+{
+	auto view = Engine::Registry().view<BallGlowing, Sprite, SimpleCollision>();
+	for (auto entity : view) {
+		
+		auto& glow = view.get<BallGlowing>(entity);
+		auto& sprite = view.get<Sprite>(entity);
+		auto& collision = view.get<SimpleCollision>(entity);
+		static bool change = false;
+		if (collision.colided)
+		{
+			glow.isColored = true;
+			collision.colided = false;
+		}
+
+		if (glow.isColored)
+		{
+			if (change) change = false;
```

Comment:
try to use {} all the time, no matter how many lines you have.

Reply:
Okay, I will